pandas_access/__init__.py

```diff
@@ -39,7 +40,14 @@ def list_tables(rdb_file, encoding="latin-1"):
         actually be UTF-8.
     :return: A list of the tables in a given database.
     """
-    tables = subprocess.check_output([mdb_tables, rdb_file]).decode(encoding)
+    if sys.platform == 'win32':
+        si = subprocess.STARTUPINFO()
+        si.dwFlags = subprocess.STARTF_USESHOWWINDOW
+        si.wShowWindow = subprocess.SW_HIDE
```

Comment:
Do these flags cause problems on non-Windows? Even if they're not needed, it's nice to reduce the platform specific code where possible.

Reply:
Unfortunately, none of these are defined on non-Windows. That would be too convenient.

Reply:
Ah, of course